manifests/cli/config.pp

```diff
@@ -24,8 +24,10 @@
   validate_string($ssh_private_key_content)
 
   # required by PuppetX::Jenkins::Provider::Clihelper base
-  package { 'retries':
-    provider => 'gem',
+  if ! defined(Package['retries']) {
```

Comment:
We've been making these changes elsewhere, it's probably better to just use [ensure_packages](https://github.com/puppetlabs/puppetlabs-stdlib#ensure_packages) for this.

If you've not got the time, I can update the changeset tomorrow